Running-Mastodon/PgBouncer-guide.md

```diff
@@ -87,7 +78,7 @@ listen_addr = 127.0.0.1
 listen_port = 6432
 ```
 
-Put `md5` as the `auth_type` (assuming you're using the md5 format in `userlist.txt`):
+Put `md5` as the `auth_type`:
```

Comment:
Just saw that the docs say about `md5`: ["`auth_file` may contain both MD5-encrypted or plain-text passwords"](https://pgbouncer.github.io/config.html#authtype) which makes sense now. May be worth mentioning in case others find it confusing.